docs/sftt/sftt-api-spec.md

```diff
@@ -252,23 +252,74 @@ No request body.
 
 ##### `200 OK`
 
-!!! missing "JSON body"
+```json
+{
+  "id": INT,
+  "name": STRING,
+  "bio": STRING,
+  "members": [
+    {
+      "user_id": INT,
+      "username": STRING,
+      "team_role": STRING,
+    },
+    ...
+  ],
+  "goals": [
+    {
+      "id": INT,
+      "goal": INT,
+      "progress": INT,
+      "start_date": TIMESTAMP,
+      "complete_by": TIMESTAMP,
+      "completion_date": TIMESTAMP
+    },
+    ...
+  ]
+}
+```
+
+##### `400 BAD REQUEST`
+
+If the team id is invalid.
 
 ### Add a Goal
 
 `POST api/v1/protected/teams/:team_id/add_goal`
 
 Adds a goal to this team's list of goals.
 
-!!! missing "This route has not been implemented yet"
+#### Request Body
+
+```json
+{
+  "goal": INT,
+  "start_at": TIMESTAMP,
+  "complete_by": TIMESTAMP
+}
+```
+
+##### `400 BAD REQUEST`
+
+If the goal is negative or the `complete_by` date is before the `start_at` date.
```

Comment:
There should definitely be a 401 response for this route too, who's allowed to add goals?

Reply:
Fair point, I'll restrict it to team leaders